### DISCUSSION TITLE
Games that inspire

### DISCUSSION THREAD

**Discussion by Jackcarlisle1** [jeffaroundtheworld/2024gamecollab#4] [General]
What games inspire you for this project?

**Comment by kodriq** [comment]
we can edit each others messages i think so we can all just add to this list

- portal
- bloodborne
- the forest
- poppy playtime
- SCP
- slenderman
- doors roblox 💀 bahahahahaa
- little nightmares
- Ao Oni
- among us
- baldis basics
- postal 2
